src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -1,13 +1,67 @@
 package wolox.training.controllers;
 
-import org.springframework.stereotype.Controller;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
 
-@Controller
+@RestController
+@RequestMapping("/api/books")
 public class BookController {
 
+
+    @Autowired
+    private BookRepository bookRepository;
+
+    /**
```

Comment:
One of the most important things about javadocs is the explanation of the method being documented.
That description would be missing, indicating the purpose of the method in each of the documented methods.